lib/web_ui/dev/felt

```diff
@@ -44,4 +44,4 @@ then
   ninja -C $HOST_DEBUG_UNOPT_DIR
 fi
 
-(cd $WEB_UI_DIR && $DART_SDK_DIR/bin/dart dev/felt.dart $@)
+($DART_SDK_DIR/bin/dart "$DEV_DIR/felt.dart" $@)
```

Comment:
Can you please add `set -e` at the top of the script so it doesn't keep on truckin' when intermediate commands fail?